Gruntfile.js

```diff
@@ -3,6 +3,11 @@ module.exports = function(grunt) {
   // The target directory for the final build product.
   var targetDirectory = 'out';
 
+  // Closure dep paths
+  var closureLibraryPath = ['node_modules', 'google-closure-library'].join('/')
```

Comment:
This is actually [hard coded](https://github.com/nodejs/node-v0.x-archive/blob/master/lib/module.js#L215) into node, and [convention](http://stackoverflow.com/questions/21818701/use-a-custom-directory-name-instead-of-node-modules-when-installing-with-npm), so should be good here.